CMakeLists.txt

```diff
@@ -80,6 +75,14 @@ file(GLOB SOURCES "src/*.cpp")
 
 add_library(demucs.cpp.lib ${SOURCES})
 
+# add library Eigen3
+# expose source files to project linking to this one
+target_include_directories(demucs.cpp.lib PUBLIC
```

Comment:
The change looks good to me, I just researched it a little bit.

Would `PRIVATE` be more appropriate than `PUBLIC`?

* I don't really want or care for libnyquist being exposed to end users. Actually libnyquist should only be used for decoding media files in the cli programs, not in the demucs inference library itself
* Eigen definitely should be exposed to end users, since my api anticipates Eigen-like inputs

In #9 (general upkeep PR), I'll try to separate libnyquist from demucs.cpp.lib